core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -77,11 +87,19 @@ oppia.factory('CollectionNodeObjectFactory', [
     // frontend exploration summary tile displaying. Changes to the returned
     // object are not reflected in this domain object.
     CollectionNode.prototype.getExplorationSummaryObject = function() {
-      // TODO(bhenning): This should be represented by a frontend summary tile
-      // domain object that is also shared with the exploration editor/viewer.
+      // TODO(bhenning): This should be represented by a frontend summary domain
+      // object that is also shared with the exploration editor/viewer.
```

Comment:
Why would the exploration editor/viewer use this? It currently doesn't, since it already has the full exploration...